lib/repo/repository.go

```diff
@@ -18,6 +27,13 @@ import (
 
 // S3Bucket keeps default S3 bucket path
 const S3Bucket = "https://cdn.isaax.io/isaax-distro/versions.json"
+const IoTItRepo = "https://cdn.isaax.io/iotit/version.json"
```

Comment:
__[golint]__
* [golint] exported const IoTItRepo should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->